app/assets/reset.css

```diff
@@ -92,6 +92,7 @@ textarea {
   border: 0;
   font-family: inherit;
   font-size: 100%;
+  font-weight: unset;
```

Comment:
Is this a custom addition from us to the reset?

Reply:
Yes, I used the same CSS reset which @juliendonck used for other projects. But since we're using `<h1>`, `<h2>` and `<h3>` for our headings we had to add this custom rule.

Otherwise the html elements have different font weights than the ones in our style guide.